### DISCUSSION TITLE
Custom view Preview back button

### DISCUSSION THREAD

**Discussion by Read2uk** [Laravel-Backpack/community-forum#1207] [Q&A (Help)]
Hi,

Just a question, I realized when you preview a record on a Custom View (not the default one), you have the back button, it takes you back to default view rather than the custom view where the record was previewed.

Is this intended behaviour?

**Comment by karandatwani92** [comment]
Hey @Read2uk 

Thanks for highlighting this. It is something that would be better for the user's experience. I have escalated the issue for the same here https://github.com/Laravel-Backpack/PRO/issues/290

Thanks for reporting.
